worlds/dark_souls_3/Items.py

```diff
@@ -1,17 +1,32 @@
+import sys
+
 from BaseClasses import Item
-from worlds.dark_souls_3.data.items_data import item_tables
+from worlds.dark_souls_3.data.items_data import item_tables, dlc_shields_table, dlc_weapons_upgrade_10_table, \
+    dlc_weapons_upgrade_5_table, dlc_goods_table, dlc_spells_table, dlc_armor_table, dlc_ring_table, dlc_misc_table
 
 
 class DarkSouls3Item(Item):
     game: str = "Dark Souls III"
 
+    dlc_set = {**dlc_shields_table, **dlc_weapons_upgrade_10_table, **dlc_weapons_upgrade_5_table,
+               **dlc_goods_table, **dlc_spells_table, **dlc_armor_table, **dlc_ring_table, **dlc_misc_table}
+
     @staticmethod
     def get_name_to_id() -> dict:
         base_id = 100000
         table_offset = 100
 
         output = {}
         for i, table in enumerate(item_tables):
+            if len(table) > table_offset:
+                print("An item table has {} entries, that is more than {} entries ({})".format(len(table), table_offset, i))
+                sys.exit(1)
             output.update({name: id for id, name in enumerate(table, base_id + (table_offset * i))})
 
         return output
+
+    @staticmethod
```

Comment:
why not make this a property on DarkSouls3Item?

Reply:
im not too sure on marechs reasoning on this, as i thought we changed that once already to be a DLC list.

Reply:
Because it shouldn't be called from an instance of DarkSouls3Item.
This method is used to create the item_name_to_id dict which is in the scope of the DarkSouls3Item class but not in the scope of an individual instance of DarkSouls3Item.

Reply:
the comment is attached to "is_dlc_item"